InOffice/HomeScreen/HomeViewController.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  HomeViewController.swift
+//  InOffice
+//
+//  Created by Luka Muše on 24.02.2022..
+//
+
+import UIKit
+
+
+class HomeViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    private var shouldResize: Bool?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupViews()
+        setupBackground()
+        view.backgroundColor = .white
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        showImage(false)
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        showImage(true)
+    }
+    /*
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        guard let shouldResize = shouldResize
+        else { assertionFailure("shouldResize wasn't set. reason could be non-handled device orientation state"); return }
+        
+        if shouldResize {
+            moveAndResizeImageForPortrait()
+        }
+    }
+    
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        guard let shouldResize = shouldResize
+        else { assertionFailure("shouldResize wasn't set. reason could be non-handled device orientation state"); return }
+        
+        if shouldResize {
+            moveAndResizeImageForPortrait()
+        }
+    }
+*/
+
+    private var imageView: UIImageView = {
+        let image = UIImageView(image: UIImage(named: "user"))
+        //let tapImage = UITapGestureRecognizer(target: self, action: #selector(imageTapped(tapImage:)))
+        //image.isUserInteractionEnabled = true
+        //image.addGestureRecognizer(tapImage)
+        return image
```

Comment:
nemoj commitat ovo ako nece se nastavljat... radije stavi komentar // TODO: Create an action for image